### ISSUE TITLE
Setting 'taken date' for videos is not working

### ISSUE THREAD

**Issue by oPromessa** [oPromessa/flickr-uploader#9]
An error on video_date... checking.

**Comment by oPromessa** [comment]
Possibly due to a prior error in uploading operation. 
Nope... it's an error from flickr..

```
(<class 'requests.exceptions.ConnectionError'>, ConnectionError(ProtocolError('Connection aborted.', BadStatusLine("''",)),), <traceback object at 0x7fe175fdafc8>)
Error setting date
```

Current error ration 1 out of 291... attempts.

Two options:
- make several set dates attempts
- log it and update it on the next upload round !!!

**Comment by oPromessa** [comment]
Solved on - Version [2.5.7](https://github.com/oPromessa/flickr-uploader/tree/2.5.7)
